src/Offer/CommandHandlers/UpdatePriceInfoHandler.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Offer\Commands\UpdatePriceInfo;
+use CultuurNet\UDB3\Offer\OfferRepository;
+
+class UpdatePriceInfoHandler implements CommandHandler
+{
+    private OfferRepository $offerRepository;
+
+    public function __construct(OfferRepository $offerRepository)
+    {
+        $this->offerRepository = $offerRepository;
+    }
+
+    public function handle($command): void
+    {
+        if (!($command instanceof UpdatePriceInfo)) {
+            return;
+        }
+
+        $offer = $this->offerRepository->load($command->getItemId());
+        $offer->updatePriceInfo(
```

Comment:
Here a conversion could happen from the Model PriceInfo to the Legacy PriceInfo

Reply:
Same as previous comment, I think we shouldn't make this PR too big.